src/style.css

```diff
@@ -87,5 +107,12 @@ button {
 .material-symbols-outlined {
   font-size: 2rem;
   color: gray;
+}
+
+.header .material-symbols-outlined {
   cursor: pointer;
 }
+
+/* .over {
+  border: 3px dotted #666;
+} */
```

Comment:
- [ OPTIONAL ] Why dont you want to remove commented code from your project to make your code clean? :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)